examples/tutorials/10_Exploring_Quantum_Chemistry_with_GDB1k.ipynb

```diff
@@ -342,19 +390,20 @@
         "colab": {}
       },
       "source": [
-        "# def rf_model_builder(model_params, model_dir):\n",
-        "#   sklearn_model = RandomForestRegressor(**model_params)\n",
-        "#   return dc.models.SklearnModel(sklearn_model, model_dir)\n",
-        "# params_dict = {\n",
-        "#     \"n_estimators\": [10, 100],\n",
-        "#     \"max_features\": [\"auto\", \"sqrt\", \"log2\", None],\n",
-        "# }\n",
+        "def rf_model_builder(n_estimators, max_features, model_dir):\n",
+        "  sklearn_model = RandomForestRegressor(n_estimators=n_estimators, \n",
+        "                                        max_features=max_features)\n",
+        "  return dc.models.SklearnModel(sklearn_model, model_dir)\n",
+        "params_dict = {\n",
+        "    \"n_estimators\": [10, 100],\n",
+        "    \"max_features\": [\"auto\", \"sqrt\", \"log2\", None],\n",
```

Comment:
I think basically `n_estimators` value is bigger than these values (100~500 is usual) and we don't perform `max_features` tuning.